working-groups.md

```diff
@@ -385,3 +386,25 @@ and a place to discuss community operations and process.
 | <img width="30px" src="https://github.com/bobcatfish.png"> | Christie Wilson   | Google    | [bobcatfish](https://github.com/bobcatfish) |
 | <img width="30px" src="https://github.com/dibyom.png">     | Dibyo Mukherjee   | Google    | [dibyom](https://github.com/dibyom)         |
 | <img width="30px" src="https://github.com/vdemeester.png"> | Vincent Demeester | Red Hat   | [vdemeester](https://github.com/vdemeester) |
+
+## Software Supply Chain Security (s3c)
+
+The goal of this working group is to discuss supply chain security initiatives across Tekton (exact scope 
+TBD [community#629](https://github.com/tektoncd/community/issues/629)).
+
+| Artifact                   | Link                       |
+| -------------------------- | -------------------------- |
+| Forum                      | [tekton-dev@](https://groups.google.com/forum/#!forum/tekton-dev) |
+| Meeting Notes              | [Discussion board](https://github.com/orgs/tektoncd/projects/14/views/1), [HackMD Notes](https://hackmd.io/gFcAZMFMRwuTaZ1i7Y3fSg)
+| Slack Channel              | [#security](https://tektoncd.slack.com/messages/security) |
+| Community Meeting VC       | [meet.google.com/utk-kdff-hci](https://meet.google.com/utk-kdff-hci) |
```

Comment:
I setup a zoom meeting as discussed https://zoom.us/j/96593435267?pwd=TTNVYUJEQlNzMXlKYjFXcUwzOUZEdz09

Reply:
See https://github.com/tektoncd/community/issues/630

Reply:
Thanks @afrittoli ! I've updated the link here